calendar-bundle/contao/modules/ModuleEventlist.php

```diff
@@ -328,8 +336,17 @@ protected function compile()
 			{
 				$objTemplate->header = true;
 				$strDate = $event['firstDate'];
+				$eventIndex = 0;
+				++$groupIndex;
 			}
 
+			$groupCounts[$groupIndex] = ($groupCounts[$groupIndex] ?? 0) + 1;
+
+			$objTemplate->groupIndex = $eventIndex++;
+			$objTemplate->groupCount = static function () use ($groupIndex, &$groupCounts): int {
+				return $groupCounts[$groupIndex];
+			};
```

Comment:
```suggestion
			$objTemplate->groupCount = static fn (): int => $groupCounts[$groupIndex];
```

Reply:
Does that use `$groupCounts` as a reference?